content/how-to/account-management/_index.md

```diff
@@ -16,10 +16,10 @@ managing invoices, and more.
 
 ### Account Management
 
-- [Change an account's company name](/support/how-to/change-account-company-name)
+- [Change your Account Company Name](/docs/portal-onboarding-guide/administrative/change_account_company_name#)
```

Comment:
we will leave the link/domain `/support/how-to/change-account-company-name` on this line. But we have to find it in the `_redirects` file, on line:120 i believe. Then update the ending of line:120 to say `change_account_company_name` since we need this new name.